bin/pycbc_live

```diff
@@ -1116,18 +1129,18 @@ with ctx:
         maxlen = args.max_length
     maxlen = int(maxlen)
     data_reader = {ifo: StrainBuffer.from_cli(ifo, args, maxlen)
-                   for ifo in ifos}
```

Comment:
I am thinking that this should be `evnt.ifos` for rank 0, and `evnt.trigg_ifos` for the other ranks (where there is no point wasting time and CPU reading and conditioning the data for the localization-only detectors).

Reply:
Definitely agree

Reply:
This is now implemented.